AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -520,14 +510,11 @@ + (void)didEnterBackground:(NSNotification *)notification
     // the app is resumed.  Non-visible controllers can be more aggressively culled to the LowMemory state (see definitions for documentation)
     BOOL isVisible = ASInterfaceStateIncludesVisible([rangeController interfaceState]);
     [rangeController updateCurrentRangeWithMode:isVisible ? ASLayoutRangeModeVisibleOnly : ASLayoutRangeModeLowMemory];
+    // N.b. we intentionally don't call `performRangeUpdate` because `updateCurrentRangeWithMode` does so if needed.
   }
   
   // Because -interfaceState checks __ApplicationState and always clears the "visible" bit if Backgrounded, we must set this after updating the range mode.
   __ApplicationState = UIApplicationStateBackground;
-  for (ASRangeController *rangeController in allRangeControllers) {
-    // Trigger a range update immediately, as we may not be allowed by the system to run the update block scheduled by changing range mode.
-    [rangeController performRangeUpdate];
```

Comment:
I am nearly certain this is necessary because the value of __ApplicationState must be updated / changes the behavior of the range update